package.json

```diff
@@ -10,6 +10,7 @@
     "gl-shader": "^4.2.1",
     "gl-vec3": "^1.0.0",
     "glsl-inverse": "^1.0.0",
+    "glsl-out-of-range": "^1.0.2",
```

Comment:
Nice https://www.npmjs.com/package/glsl-out-of-range is alive !

Now, could add it to the plotly npm org, so that anyone working for us can publish changes. 

To do so, see -> https://github.com/plotly/dev-docs/blob/master/advanced/publish-to-npm.md#add-new-module-to-plotly-npm-org

Reply:
I added using the link. Thanks.